webapp/portlet/src/main/webapp/vue-apps/common/components/ExoUserAvatar.vue

```diff
@@ -12,7 +12,8 @@
       :class="avatarClass">
       <v-avatar
         :size="size"
-        class="ma-0 pull-left">
+        :class="pullLeft"
```

Comment:
Does this will change behaviour from ExoUserAvatar when used with mobile in another component than ActivityHead ?

Reply:
pull left won't be applied when 'this.isMobile' 
`    pullLeft() {
      return this.isMobile && ' ' || 'pull-left';
    },
`

Reply:
Before your change, pull-left is always applied. 
After your change, it will not be applied when in mobile.

Does it will change something when the component ExoUserAvatar is used in another component then ActivityHead ?

Reply:
thank you for the clarification,
no the exoUserAvatar remained with same behaviour, it wont be affected with change .